proto/agent_command_test.go

```diff
@@ -26,7 +26,7 @@ func (s *CommandTestSuite) SetupTest() {
 			HostID:     "host",
 			HostSecret: "secret",
 			StatusPort: 2286,
-			LogPrefix:  "prefix",
+			LogPrefix:  "LOCAL",
```

Comment:
Should these use evergreen.LocalLoggingOverride?

Reply:
Yeah, they should, thanks. I'll push this separately.